code/workspaces/infrastructure-api/src/kubernetes/serviceAccountApi.spec.js

```diff
@@ -16,9 +17,12 @@ describe('createNamespacedServiceAccount', () => {
   it('calls k8s api to create a namespaced service account', async () => {
```

Comment:
Is it worth having two tests, one that checks it makes the Service Account if it doesn't exist and one to make sure it doesn't try to create it if it does exist?